src/components/Drawer/Drawer.styles.js

```diff
@@ -5,7 +5,7 @@ import spacing from "../../theme/spacing";
 import getThemeValue from "../../utils/getThemeValue";
 
 export const DrawerContainer = styled.div`
-  background-color: ${getThemeValue("onyx", "base")};
+  background-color: ${getThemeValue("dark", "base")};
```

Comment:
Is this a desired global change or should this be scoped behind a prop (due to a feature flag)?

Reply:
According to the discussion with Siahey Fedarovich this is appropriate global color